.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
```

Comment:
what does this workflow do?

Reply:
Yeah, right now that's the only thing we have that really releases anything, heh, but anything release related could be in here.

IE releasing crates to cargo.